src/dictionary/infrastructure/WordRepository.test.ts

```diff
@@ -1,11 +1,14 @@
 import Promise from 'bluebird'
 import { testDelegation, TestData } from 'test-support/utils'
 import WordRepository from './WordRepository'
+import ApiClient from 'http/ApiClient'
+
+const apiClient = new (ApiClient as jest.Mock<ApiClient>)()
+const mockFetchJson = jest.fn()
+const mockPostJson = jest.fn()
+apiClient.fetchJson = mockFetchJson
+apiClient.postJson = mockPostJson
```

Comment:
Set up an automatic mock and use correct types.

Reply:
i can right click run it on webstorm otherwise it doesn't work.